README.md

```diff
@@ -134,7 +134,7 @@ Complete all of these tasks on your local home machine.
     you can skip this section.
   * Amazon EC2
 
-        sudo pip install boto
```

Comment:
In my testing **both** `boto` and `boto3` are required. Can you change this to: `sudo pip install boto boto3`?